public/javascripts/payments.js

```diff
@@ -776,6 +870,16 @@
     selector.querySelector(`option[value=${country}]`).selected = 'selected';
     selector.className = `field ${country.toLowerCase()}`;
 
+    //update currency if there's a currency for that country
+    switch(country){
+      case 'AU':
+        activeCurrency = 'aud';
+        break;
+      default:
+        activeCurrency = config.currency;
+        break;
+    }
```

Comment:
Until there are more cases that need these, I think it's more idiomatic to use a regular `if` statement.

Reply:
Thanks, I've updated this to an if statement.